tox.ini

```diff
@@ -0,0 +1,58 @@
+[tox]
+envlist =
+    flake8
+    isort
+    py{27,34,35,36}-dj111-cms35
```

Comment:
Please add here all supported django versions not only dj111

Reply:
I added a comment about versions because we have it in our compatibility table.

I'm good with keeping only dj111 here. 
Do we have somewhere information that versions below 1.11 do not fully support by cms 3.5.3?

Reply:
This is the compatibility table: http://docs.django-cms.org/en/latest/

I'd only keep 1.11 in this package, the rest is tested in django-cms itself.

Reply:
👍